src/V3Timing.h

```diff
@@ -0,0 +1,40 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+// DESCRIPTION: Verilator: Prepare AST for features features
+//
+// Code available from: https://verilator.org
+//
+//*************************************************************************
+//
+// Copyright 2003-2022 by Wilson Snyder. This program is free software; you
+// can redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+
+#ifndef _V3TIMING_H_
+#define _V3TIMING_H_
+
+#include "config_build.h"
+#include "verilatedos.h"
+
+#include "V3Ast.h"
+
+//============================================================================
+
+class V3Timing final {
+public:
+    // Statements to inject into the global eval function
+    struct TimingKit {
+        std::map<const AstVarScope*, AstSenTree*> varToSenTreep;  // Additional var sensitivities
+        AstNodeStmt* preStmts = nullptr;  // Statements to put before the active eval loop
+        AstNodeStmt* postStmts = nullptr;  // Statements to put after the active eval loop
```

Comment:
I think at least some of these comments are incorrect. I think `preStms` currently go before trigger computation, and `postStmts` go inside the loop, but after the main logic. I think this will need to change a bit though, so maybe don't worry too much about it.

Reply:
No longer applicable